recipes/jungle/all/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, CMake, tools, RunEnvironment
+from conan.tools.build import cross_building
+import os
+import subprocess
+import re
```

Comment:
```suggestion
from conan import ConanFile
from conan.tools.build import cross_building
from conans import CMake
import os
```
No she-bang. Avoid unused imports